specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2020-06-14/examples/KustoAttachedDatabaseConfigurationsCreateOrUpdate.json

```diff
@@ -1,61 +1,61 @@
-          "databaseName": "db1",
-          "defaultPrincipalsModificationKind": "Union",
-          "provisioningState": "Creating"
-        }
-      }
-    }
-  }
-}
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-123456789098",
+    "resourceGroupName": "kustorptest",
+    "clusterName": "kustoclusterrptest4",
+    "attachedDatabaseConfigurationName": "attachedDatabaseConfigurations1",
+    "api-version": "2020-06-14",
+    "parameters": {
+      "location": "westus",
+      "properties": {
+        "clusterResourceId": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Kusto/Clusters/KustoClusterLeader",
+        "databaseName": "kustodatabase",
+        "defaultPrincipalsModificationKind": "Union"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Kusto/Clusters/KustoClusterRPTest4/attachedDatabaseConfigurations/attachedDatabaseConfigurations1",
+        "name": "KustoClusterRPTest4/attachedDatabaseConfigurations1",
+        "type": "Microsoft.Kusto/Clusters/AttachedDatabaseConfigurations",
+        "location": "westus",
+        "properties": {
+          "clusterResourceId": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Kusto/Clusters/KustoClusterLeader",
+          "databaseName": "db1",
+          "defaultPrincipalsModificationKind": "Union",
+          "provisioningState": "Succeeded"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Kusto/Clusters/KustoClusterRPTest4/attachedDatabaseConfigurations/attachedDatabaseConfigurations1",
+        "name": "KustoClusterRPTest4/attachedDatabaseConfigurations1",
+        "type": "Microsoft.Kusto/Clusters/AttachedDatabaseConfigurations",
+        "location": "westus",
+        "properties": {
+          "clusterResourceId": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Kusto/Clusters/KustoClusterLeader",
+          "databaseName": "db1",
+          "defaultPrincipalsModificationKind": "Union",
+          "provisioningState": "Creating"
+        }
+      }
+    },
+    "202": {
+      "body": {
+        "id": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Kusto/Clusters/KustoClusterRPTest4/attachedDatabaseConfigurations/attachedDatabaseConfigurations1",
+        "name": "KustoClusterRPTest4/attachedDatabaseConfigurations1",
+        "type": "Microsoft.Kusto/Clusters/AttachedDatabaseConfigurations",
+        "location": "westus",
+        "properties": {
```

Comment:
Can you revert your changes to all these unrelated files . This will mess with the history for all of these.